source/_patterns/10-atoms/navigation/search.scss

```diff
@@ -189,23 +157,56 @@
   }
 }
 
-/* footer variant */
-.searchbox.search--footer {
+.searchbox--navigation {
   position: relative;
   width: 100%;
   top: auto;
   right: auto;
   max-width: 458px;
   margin: auto;
-  padding:  13px 24px 0;
+  padding: 0 12px;
 
   @include desktop-up {
     max-width: 500px;
   }
+
+  @include tablet-up {
+    margin-bottom: 32px;
+  }
+
+  .searchbox__input {
+    height: 40px;
+    background-size: auto;
+  }
 }
 
-.search--footer .searchbox__input {
-  height: 40px;
-  background-size: auto;
-  background-position: auto;
+// Inside of the header navigation, the searchbox should only be visible on Size 1 (Mobile)
+.searchbox--header-navigation {
+  margin-top: 16px;
+
+  @include tablet-up {
+    display: none;
+  }
+}
+
+// Expandable searchboxes have a fixed initial with and can, surprisingly, expand to 100%
+.searchbox--expandable {
+  float: right;
+  width: 180px;
+  transition: width .2s ease;
+}
+
+.searchbox--header {
+  .searchbox__input {
```

Comment:
.searchbox--header__input instead of .searchbox__input, but I'm not sure about this

Reply:
`--header` is the modifier